spec/requests/api/v1/subscriptions_spec.rb

```diff
@@ -51,6 +51,105 @@
     end
   end
 
+  describe 'create_with_override' do
+    let(:billable_metric) { create(:billable_metric, organization: organization) }
+    let(:standard_charge) { create(:standard_charge, billable_metric: billable_metric) }
+    let(:graduated_charge) { create(:graduated_charge, billable_metric: billable_metric) }
+
+    let(:plan) do
+      create(:plan, organization: organization, charges: [standard_charge, graduated_charge])
+    end
+
+    let(:params) do
+      {
+        external_customer_id: customer.external_id,
+        name: 'subscription name',
+        overridden_plan_code: plan.code,
+        external_id: SecureRandom.uuid,
+        billing_time: 'anniversary',
+        plan: {
+          amount_cents: 100,
+          amount_currency: 'EUR',
+          trial_period: 1,
+          charges: [
+            {
+              id: standard_charge.id,
+              charge_model: 'standard',
+              properties: {
+                amount: '0.22',
+              },
+            },
+            {
+              id: graduated_charge.id,
+              charge_model: 'graduated',
+              properties: [
+                {
+                  to_value: 1,
+                  from_value: 0,
+                  flat_amount: '0',
+                  per_unit_amount: '0',
+                },
+                {
+                  to_value: nil,
+                  from_value: 2,
+                  flat_amount: '0',
+                  per_unit_amount: '3200',
+                },
+              ],
+            },
+          ],
+        }
+      }
+    end
+
+    before { plan }
+
+    it 'returns a success' do
+      post_with_token(organization, '/api/v1/subscriptions/override', { subscription: params })
+
+      expect(response).to have_http_status(200)
```

Comment:
RSpec/Rails/HttpStatus: Prefer `:ok` over `200` to describe HTTP status code. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Rails/HttpStatus)